contracts/sol6/KyberMatchingEngine.sol

```diff
@@ -204,6 +209,17 @@ contract KyberMatchingEngine is KyberHintHandler, IKyberMatchingEngine, Withdraw
         (reserveAddress, , , ,) = kyberStorage.getReserveDetailsById(reserveId);
     }
 
+    function areAllReservesListed(
+        bytes32[] memory reserveIds,
+        IERC20 src,
+        IERC20 dest
+    ) internal override view returns (bool) {
```

Comment:
please use boolean values

returns(bool result)
and don't declare another one.

Reply:
Fixed